Jenkinsfile-build

```diff
@@ -0,0 +1,34 @@
+pipeline {
+  agent none
+
+  stages {
+    stage('Test') {
+        agent {
+            ecs {
+                inheritFrom 'ecs'
+            }
+        }
+        steps {
+            checkout scm
+            sh 'sbt dist'
+            stash includes: 'Dockerfile', name: 'Dockerfile'
+            stash includes: 'target/universal/tdr-transfer-frontend-1.0-SNAPSHOT.zip', name: 'tdr-transfer-frontend-1.0-SNAPSHOT.zip'
```

Comment:
Today I learned about `stash includes`. :+1: 

Can we use a wildcard so it doesn't depend on a specific version number? Ditto for the `unstash` step below.

Reply:
The includes can use a wildcard which I've changed. The name can be anything you like which I didn't know so I can call it something without the version and reference that in the unstash step.